### ISSUE TITLE
aria-required should be allowed on input type="file"

### ISSUE THREAD

**Issue by juliemoynat-tanaguru** [validator/validator#995]
Hi,

The [HTML validator](https://validator.w3.org/nu/#textarea) is telling me that using an `aria-required` attribute on an `<input type="file">` is an error. In the [ARIA in HTML specification](https://w3c.github.io/html-aria/#el-input-file), I can read that this field can accept all "[Global aria-* attributes](https://w3c.github.io/html-aria/#index-aria-global)". `aria-required` is not on the list.

But, if I read the [aria-required specification](https://w3c.github.io/html-aria/#att-required), I can read "Use the `aria-required` attribute on any element that is allowed the `required` attribute in HTML."

I can use a `required` attribute on an `<input type="file">` so I should be able to use an `aria-required` attribute too.

It shouldn't be an error in the validator.

(For your information, I discussed that point about on [ARIA in HTML specification Github](https://github.com/w3c/html-aria/issues/234) at first but it appears that this is not a specification problem.)

**Comment by sideshowbarker** [comment]
Thanks very much for taking time to report this. I’ve fixed in the sources and pushed the fix to https://validator.w3.org/nu/
